src/commands/index.ts

```
@@ -0,0 +1,766 @@
+                const confirmationMessage = `📋 **Ticket Summary Preview:**\n\n` +
+                    `━━━━━━━━━━━━━━\n` +
+                    `"${userState.ticket.summary}"\n` +
+                    `━━━━━━━━━━━━━━\n\n` +
+                    `❓ **Is this description complete?**\n\n` +
+                    `• **Yes**: Proceed to the next step\n` +
+                    `• **No**: Revise your description`;
+                
+                const confirmationReply = await safeReply(ctx, confirmationMessage, {
+                    parse_mode: 'Markdown',
+                    ...Markup.inlineKeyboard([
+                        [
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)